src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.html

```diff
@@ -1,21 +1,16 @@
-<div class="example-container usa-grid">
-  <ng-container *ngTemplateOutlet="template"></ng-container>
+<div class="example-container usa-grid" style="max-height: 400px; overflow-y: scroll;">
+  <ng-container *ngTemplateOutlet="gridTemplate"></ng-container>
 </div>
 
-<ng-template #template>
-  <!-- replace div for form group <div [formGroup]="selected"> -->
+<ng-template #gridTemplate>
   <div>
-    <sam-datatable #table samSort [dataSource]="samTableDataSource" (keyup)="handleKeyup($event)">
-
+    <sam-datatable #table samSort [dataSource]="hierarchicalDataSource">
       <ng-container cdkColumnDef="select">
         <th sam-header-cell *cdkHeaderCellDef sam-sort-header>Select</th>
         <td sam-cell *cdkCellDef="let row">
-          <!-- They should either check a box to srelect the key of the data or click the row and emit the key for the data in the row -->
-          <div>
-            <input type="checkbox" id="selectId" [class.selected]="isSelected(item)" (change)="onSelectItem($event,row)"
-              name="select">
-            <label for="selectId"></label>
-          </div>
+          <input type="checkbox" style="width:2em;" id="selectId" (change)="onChecked($event,row)"
```

Comment:
@yerramshilpa Same here for the css -- please add classes and move styles into styles on component